apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -840,7 +845,10 @@ export function serverSectionFromSection(section) {
     pairing_allowed: section.pairingAllowed,
     sharing_disabled: section.sharingDisabled,
     course_id: section.courseId,
+    // TODO: (madelynkasula) remove duplicated script/script_id params once
+    // both section create/update dashboard actions use strong params
     script: (section.scriptId ? {id: section.scriptId} : undefined),
+    script_id: section.scriptId,
```

Comment:
🏅 nice work continuing to support the old format while you wait for the server changes to launch